boa_tester/src/results.rs

```diff
@@ -264,7 +264,7 @@ pub(crate) fn compare_results(base: &Path, new: &Path, markdown: bool) {
             format!(
                 "{}{}{}{}",
                 if diff == 0 { "" } else { "**" },
-                if diff > 0 { "+" } else { "" },
+                if diff.is_positive() { "+" } else { "-" },
```

Comment:
Think this resulted in the `-0`s,  swapping the logic might be a good idea? Check if it's negative, and if so use `-`, this way we get `+0` instead of `-0`.
```suggestion
                if diff.is_negative() { "-" } else { "+" },
```

Reply:
I added an extra check, and that way 0 will not be positive or negative :)

Reply:
@RageKnify let me know if the new solution is OK and we can merge this :)